src/OpenFeature.Contrib.Providers.EnvVar/README.md

```diff
@@ -0,0 +1,42 @@
+# .NET Environment Variable Provider
+
+This provider supports using the OpenFeature SDK to evaluate feature flags backed by environment variables.
+
+## Installation
+
+### .NET CLI
+
+```shell
+dotnet add package OpenFeature.Contrib.Providers.EnvVar
+```
+
+## Using the ConfigCat Provider with the OpenFeature SDK
+
+The following example shows how to use the Environment Variable provider with the OpenFeature SDK.
+
+```csharp
+using System;
+using ConfigCat.Client;
```

Comment:
Can this be removed? I'm assuming ConfigCat isn't required. Perhaps it's a copy/paste issue.